docker-osm/indonesia-buildings/docker-compose.yml

```diff
@@ -113,31 +114,31 @@ services:
       - TIME=${UPDATE_TIME}
     command: bash -c "while [ ! -f /home/settings/country.pbf ] ; do sleep 1; done && python3 -u /home/download.py"
 
-  osmenrich:
-    image: kartoza/docker-osm:osmenrich-latest
-    restart: always
-    volumes:
-      - osm_data:/home/settings
-      - import_done:/home/import_done
-      - import_queue:/home/import_queue
-      - cache:/home/cache
-    links:
-      - db:db
-    environment:
-      - POSTGRES_DB=${POSTGRES_DB}
-      - POSTGRES_USER=${POSTGRES_USER}
-      - POSTGRES_PASS=${POSTGRES_PASS}
-      - POSTGRES_PORT=${PG_PORT}
-      - POSTGRES_HOST=db
-      # These are all currently the defaults but listed here for your
-      # convenience if you want to change them
-      # folder for diff which hasn't been imported yet
-      - IMPORT_QUEUE=import_queue
-      # folder for diff which has been imported
-      - IMPORT_DONE=import_done
-      # seconds between 2 executions of the script
-      # if 0, then no update will be done, only the first initial import from the PBF
-      - TIME=${UPDATE_TIME}
+#  osmenrich:
```

Comment:
I commented out osmenrich because I'm not sure about the state of the service and it's often giving errors in logs.